README.md

```diff
@@ -0,0 +1,179 @@
+# **ALTERNATE BIT PROTOCOL SIMULATOR**
+---
+## ORGANIZATION: Carleton University
+---
+### *AUTHORS:*  
+---
+### Originally Developed by	: Dr. Cristina Ruiz Martin (https://github.com/cruizm)
+### Modified by				: Rajeev Kuamr Jandhyala (https://github.com/jrajeev459) and Abhinav Achanta (https://github.com/abhinavachanta)
```

Comment:
Incorrect way of implementing hyperlinks. Please see lecture pdf and implement it.

Reply:
[author_name](guthub url)

Reply:
changes made according to project requirements and feedback by reviewers